app/javascript/lib/study-overview.js

```diff
@@ -98,7 +98,14 @@ function calculatePlotRect() {
   const numPlots = window.SCP.numPlots
 
   const height = $(window).height() - 250
-  const width = ($('#plots-tab').width() - 80) / numPlots
+
+  // Accounts for expanding "View options" after page load
+  let baseWidth = $('#plots-tab').width()
+
+  // Accounts for when "Summary" tab is selected by default
+  if (baseWidth === 0) baseWidth = $(window).width()
+
```

Comment:
I think always having ifs followed by {} is a convention we'd like to keep

Reply:
I used Jon's suggestion, which removed this line.  But I've updated `.eslintrc.js` to warn in such cases (555d1f9c).